packages/client/tests/functional/interactive-transactions/__snapshots__/tests.ts.snap

```diff
@@ -308,7 +308,13 @@ Invalid \`prisma.user.create()\` invocation in
 Unique constraint failed on the fields: (\`email\`)
 `;
 
-exports[`interactive-transactions (provider=sqlite, js_libsql) batching raw rollback 1`] = `: UNIQUE constraint failed: User.email`;
+exports[`interactive-transactions (provider=sqlite, js_libsql) batching raw rollback 1`] = `
+
+Invalid \`prisma.$executeRaw()\` invocation:
+
+
+SQLITE_CONSTRAINT_UNIQUE: UNIQUE constraint failed: User.email
+`;
```

Comment:
This now seems to be a constant failure, and I can also reproduce on my laptop.

Reply:
Do we need to create an issue for tracking this?

Reply:
We have an issue for improving errors with `libsql` I believe, I'll double check